guides/common/modules/proc_configuring-tls-for-secure-ldap.adoc

```diff
@@ -1,29 +1,30 @@
 [id="Configuring_TLS_for_Secure_LDAP_{context}"]
 = Configuring TLS for secure LDAP
 
-Use the {Project} CLI to configure TLS for secure LDAP (LDAPS).
+If you require {Project} to use TLS to establish a secure LDAP connection (LDAPS), obtain certificates used by the LDAP server you are connecting to and mark them as trusted on the base operating system of your {ProjectServer}.
+
+If your LDAP server uses a certificate chain with intermediate certificate authorities, all root and intermediate certificates in the chain must be trusted, so ensure you obtain all certificates.
 
 .Procedure
-. Obtain the Certificate from the LDAP Server.
+. Obtain the certificate from the LDAP Server.
 .. If you use Active Directory Certificate Services, export the Enterprise PKI CA Certificate using the Base-64 encoded X.509 format.
 ifndef::orcharhino[]
 See https://access.redhat.com/solutions/1498773[How to configure Active Directory authentication with `TLS` on {Project}] for information on creating and exporting a CA certificate from an Active Directory server.
 endif::[]
-.. Download the LDAP server certificate to a temporary location onto {ProjectServer} and remove it when finished.
+.. Download the LDAP server certificate to a temporary location onto {ProjectServer}, such as `/tmp/_example.crt_`.
+You will remove the certificate when finished.
```

Comment:
Not sure you need to mention this because the last step is to delete the downloaded certificate.

Reply:
I know what you mean, but I find it to be useful context for selecting the download location. No change.